phpBB/language/en/acp/extensions.php

```diff
@@ -130,4 +150,32 @@
 
 	'META_FIELD_NOT_SET'	=> 'Required meta field %s has not been set.',
 	'META_FIELD_INVALID'	=> 'Meta field %s is invalid.',
+
+	'EXTENSIONS_CATALOG_SETTINGS'	=> 'Extensions catalog settings',
+	'ENABLE_ON_INSTALL'				=> 'Enable extensions when installing',
+	'PURGE_ON_REMOVE'				=> 'Purge extensions when removing',
+	'ENABLE_PACKAGIST'				=> 'Enable Packagist usage',
+	'ENABLE_PACKAGIST_EXPLAIN'		=> 'Use packagist to browse extensions? Beware that packagist may contain extensions not validated by phpBB.',
```

Comment:
"phpBB Extension Customisations Team" would probably be more appropriate. I don't think we have anything that references a "validations team" anywhere on phpbb.com.